part_1/src/com/chernova/libraryXML/Book.java

```diff
@@ -0,0 +1,114 @@
+package com.chernova.libraryXML;
+
+import java.util.Comparator;
+import java.util.Date;
+
+public class Book implements Comparable<Book>, Comparator<Book> {
```

Comment:
It's not obvious - comparing by title (maybe by id, maybe by author and ctr.), so lets remove Comparator and Comparable both from Book